clients/include/blas1/testing_copy_batched.hpp

```diff
@@ -36,6 +36,31 @@ inline void testname_copy_batched(const Arguments& arg, std::string& name)
     hipblasCopyBatchedModel{}.test_name(arg, name);
 }
 
+template <typename T>
+void testing_copy_batched_bad_arg(const Arguments& arg)
+{
+    bool FORTRAN = arg.api == hipblas_client_api::FORTRAN;
+    auto hipblasCopyBatchedFn
+        = FORTRAN ? hipblasCopyBatched<T, true> : hipblasCopyBatched<T, false>;
+
+    hipblasLocalHandle handle(arg);
+
+    int64_t N           = 100;
+    int64_t incx        = 1;
+    int64_t incy        = 1;
+    int64_t batch_count = 2;
+
+    device_batch_vector<T> dx(N, incx, batch_count);
+    device_batch_vector<T> dy(N, incy, batch_count);
+
+    EXPECT_HIPBLAS_STATUS(hipblasCopyBatchedFn(nullptr, N, dx, incx, dy, incy, batch_count),
+                          HIPBLAS_STATUS_NOT_INITIALIZED);
+    EXPECT_HIPBLAS_STATUS(hipblasCopyBatchedFn(handle, N, nullptr, incx, dy, incy, batch_count),
+                          HIPBLAS_STATUS_INVALID_VALUE);
+    EXPECT_HIPBLAS_STATUS(hipblasCopyBatchedFn(handle, N, dx, incx, nullptr, incy, batch_count),
+                          HIPBLAS_STATUS_INVALID_VALUE);
```

Comment:
So batched is consistent....

Reply:
cuBLAS has limited batched support, no L1s are supported so these are excluded in the yaml files from cuBLAS backend.